components/composer/src/Composer.svelte

```diff
@@ -790,10 +790,10 @@
       </main>
       <footer>
         {#if _this.show_attachment_button && (id || uploadFile)}
-          <label for="file-upload" class="composer-btn file-upload">
-            <AttachmentIcon class="AttachmentIcon" />
+          <button for="file-upload" class="composer-btn file-upload">
```

Comment:
Why was this changed? If we're keeping this as a button, `for="file-upload"` should be removed since it's a label prop.